src/modules/app.js

```diff
@@ -1,6 +1,65 @@
+import NetworkGateway from './gateway.js';
+
 class Application {
-    initApp = () => {
-    }
+  constructor() {
+    this.gateWay = new NetworkGateway();
+    this.gateWay.getID('Seed');
+
+    this.inputs = document.querySelectorAll('.userdetails');
+    this.success_message = document.querySelector('.success-message');
+    this.form = document.querySelector('.data-form');
+    this.loadingAnime = document.querySelector('.loading_anime');
+    this.tableList = document.querySelector('.table-content');
+    this.refreshBtn = document.querySelector('.f5');
+  }
+
+  initApp = () => {
+    this.form.addEventListener('submit', (event) => {
+      const refInstance = event.currentTarget.ref;
+      refInstance.addUserScore();
+      event.preventDefault();
+    });
+    this.form.ref = this;
+
+    this.refreshBtn.addEventListener('click', (event) => {
```

Comment:
- [ ] Please make sure when the Refresh button is click, it should hit the GET API endpoint causing the scores list to change. Kindly Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API